### PR TITLE
Document new version

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+1.2.1
+*****
+- Bug-fixes for service version 1.0, after pilot integration of new service at BL
+
+1.2.0
+*****
+- first implementation of federal extract requirements as per november 2017
+  (service in version 1.0, extract in version 1.0.1, data model in version 1.0.1)
+- static extract implementation update according to Weisung july 1st, 2018
+- oereb lex model creation support and documentation
```
Comment (ger-benjamin) [inline CHANGES.rst]: oereb lex **version 1.1.0** model creation... ?
Or it is not useful ?

File: doc/source/changes.rst [hunk 2]
```diff
@@ -4,8 +4,60 @@ Changes/Hints for migration
...
+
+.. _changes-new-config-mapfish-print-1.2.1:
+
+MapFish Print
+"""""""""""""
+
+These are the new configuration options for the printing of your extracts:
+
+* ``display_real_estate_subunit_of_land_register``: flag whether to display the RealEstate_SubunitOfLandRegister (Grundbuchkreis)
+  in the pdf extract or not
```
Comment (ger-benjamin) [inline doc/source/changes.rst]: or not.
(missing last dot)

File: doc/source/contrib/contributing.rst [hunk 3]
```diff
@@ -295,12 +295,12 @@ Browser requests
 Once your application has started as describe above,
 you can try your oereb server as follows.
 In your browser, check the following URL:
-   * http://localhost:6543/oereb/versions
-   * http://localhost:6543/oereb/capabilities
+* http://localhost:6543/oereb/versions
+* http://localhost:6543/oereb/capabilities
```
Comment (ger-benjamin) [inline doc/source/contrib/contributing.rst]: The official path are with the fomat (json or xml):
* http://localhost:6543/oereb/versions/json
* http://localhost:6543/oereb/capabilities/json
http://localhost:6543/oereb/getegrid/json?XY=2608883,1261844

**Comment by ger-benjamin** [comment]
Should we say in the changelog that we have updated / consolidated the tests ?

File: doc/source/contrib/contributing.rst (hunk 3 above)
Comment (jwkaltz) [reply doc/source/contrib/contributing.rst]: Indeed, thanks

File: CHANGES.rst (hunk 1 above)
Comment (jwkaltz) [reply CHANGES.rst]: in my eyes, there is no real versioning of oereb lex - any custom models (including models with oereblex usage) need to conform to the model scheme used in that version of pyramid_oereb.

**Pull request merged**